docs/extensions/query-depth-limiter.md

```diff
@@ -164,3 +223,90 @@ schema.execute(
 ```
 
 </details>
+
+<details>
+  <summary>Ignoring fields with FieldRule and arguments</summary>
+
+<!-- Cannot specify python as code for this block as blacken-docs fails
+to parse the field_arguments line within FieldRule -->
```

Comment:
this is odd, I'll check it soon 😊

Reply:
I've changed that implementation so there's no need to investigate this anymore! :grin: